docs-markdown/src/test/suite/controllers/cleanup/cleanup-controller.test.ts

```diff
@@ -100,6 +118,7 @@ suite('Cleanup Controller', () => {
 		expect(spy).to.have.been.called();
 	});
 	test('cleanup repo - empty metadata - empty values', async () => {
+		await loadDocumentAndGetItReady(filePath);
```

Comment:
In debugging I noticed that some of these tests will fail when running by themselves.  This change makes these test independent of each other.